docs/src/tty.rst

```diff
@@ -33,6 +33,18 @@ Data types
           UV_TTY_MODE_IO
       } uv_tty_mode_t;
 
+.. c:type:: uv_tty_vtermstate_t
+    Console virtual terminal mode type:
+
+    ::
+
+      typedef enum {
+          /* Support xterm csi sequence */
+          UV_TTY_SUPPORTED,
+          /* Unsupport xterm csi sequence */
```

Comment:
```suggestion
          /* Don't support xterm csi sequence */
```
(tbh, I'm not 100% sure what this comment is trying to convey so perhaps this is not a good suggestion.)